.husky/pre-push

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+. "$(dirname "$0")/_/husky.sh"
+
+# Allows us to read user input below, redirects script's input to the terminal.
+exec < /dev/tty
+
+PROTECTED_BRANCH="develop"
+CURRENT_BRANCH=$(git branch --show-current)
```

Comment:
The script handles the scenario when we're on `develop` branch and pushing to it. There is also a scenario, e.g., `git push origin my-local-branch:develop`, which is not handled here. 

I tried [this approach](https://github.com/typicode/husky/issues/1052#issuecomment-943513407) to cover the use-case but something is with the input and `remote_ref` is not acquired. 

We could try more, but I believe that it's almost impossible that someone will provide `develop` explicitly as a target branch pushing from their local branch with another name (unless they really need to push to `develop`). Hence I don't think this scenario needs to be covered, but I'm happy to hear the opinions of others!